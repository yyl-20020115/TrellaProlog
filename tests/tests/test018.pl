:-initialization(main).

main :- atom_concat(X, Y, abcdef),
			write(X), write(' <==> '), write(Y), nl, fail.
main.
