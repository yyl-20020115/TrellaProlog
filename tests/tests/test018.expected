 <==> abcdef
a <==> bcdef
ab <==> cdef
abc <==> def
abcd <==> ef
abcde <==> f
abcdef <==> 
